docs/docs-content/registries-and-packs/registries/registries.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_label: "Registries"
+title: "Registries"
+description: "Learn how to add your own custom registries to Palette"
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 20
+---
+
+Palette supports different types of registries that you can add to the platform and use in your cluster profiles. You can add the following types of registries to Palette:
+
+- Open Container Initiative (OCI) Registries
+
+- Helm Registries
+
+- Legacy Pack Registries
+
+Palette supports OCI registries that support basic authentication. OCI registries can also be used to store and serve images, Helm Charts, Zarf packages, and Packs. Refer to the [OCI Registries](./oci-registry/oci-registry.md) resource to learn more about OCI registries. 
+
+## Scope
+
+Registries are added at the tenant level and are available to all users in the tenant. You can add multiple registries of the same type to Palette. If you are using a self-hosted Palette instance, or Palette VerteX, you can add registries through the system console. Registries added through the system console are available to all tenants in the system. Check out the [Self-Hosted Add Registry](../../enterprise-version/system-management/add-registry.md) guide or the [VerteX Add Registry](../../vertex/system-management/add-registry.md) guide.
+
+
+## Synchronization
+
+Registries are automatically synchronized with Palette. Helm registries are different as you can choose to disable synchronization. When you disable synchronization, you must manually synchronize the Helm registry with Palette. Refer to the [Helm Registry](helm-charts.md#synchronization-behavior) resource to learn more about the synchronization behavior of Helm registries.
+
+
+
+## Default Registries
+
+Palette comes with a set of default registries that are available to all SaaS tenants, and non-airgap self-hosted Palette environments. The default registries are listed below:
+
+| **Name** | **Provider** | **Description** | **URL** | **Base Path** |
+| --- | --- | --- | --- | --- | --- |
+| Bitnami | Helm | A Helm Chart registry containing Helm Charts maintained and supported by Bitnami. | `https://charts.bitnami.com/bitnami` | - |
+| Public Spectro Helm Repo | Helm | A Helm Chart registry containing Helm Charts maintained and supported by us. | `https://spectrocloud.github.io/helm-charts` | - |
```

Comment:
```suggestion
| Public Spectro Helm Repo | Helm | A Helm Chart registry containing Helm Charts maintained and supported by Spectro Cloud. | `https://spectrocloud.github.io/helm-charts` | - |
```
Although we are consistent in using "we" to refer to Spectro Cloud, it still felt a little jarring for me to read "supported by us". I think it's better to be specific